README.md

```diff
@@ -1,4 +1,255 @@
-# cockroach-operator
-Kubernetes Operator for CockroachDB
+# CockroachDB Kubernetes Operator
 
-This project is not production ready and is in an alpha state.
+The CockroachDB Kubernetes Operator deploys CockroachDB on a Kubernetes cluster. You can use the Operator to manage the configuration of a running CockroachDB cluster, including:
+
+- Authenticating certificates
+- Configuring resource requests and limits
+- Scaling the cluster
+- Performing a rolling upgrade
+
+## Limitations
+
+- This project is in alpha and is not production-ready.
+- The Operator currently runs on GKE.
+- Currently the Kubernetes CA is required to deploy a secure cluster.
+- Expansion of persistent volumes is not yet functional.
+
+## Prerequisites
+
+- Kubernetes 1.8 or higher
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Install the Operator
+
+<!-- Clone this repo:
+
+```
+git clone https://github.com/cockroachdb/cockroach-operator.git
+```
+
+From the `cockroach-operator` directory on your machine, create a Kubernetes cluster on GKE, specifying a cluster name (e.g., `crdb-test`).
+
+```
+./hack/create-gke-cluster.sh -c <cluster-name>
+```
+
+This creates a Kubernetes cluster on GKE, using your default GCP region and project.
+
+Install the Operator, specifying the cluster name:
+
+```
+./hack/apply-operator.sh -c <cluster-name>
```

Comment:
This is not correct anymore, please see my instructions that are in my PR.  It is two commands now.